### DISCUSSION TITLE
HRRR Reference sphere, 6370000m or 6371229m?

### DISCUSSION THREAD

**Discussion by yosukefk** [NOAA-GSL/GSL-Regional-Model-Forum#25] [Q&A] [answered]
Hello,

If I query HRRR GRIB2 file reference spheroid, it says 6371229m.  Is this correct?  I thought that WRF had hard-wired 6370000m sphere, but is my memory wrong and the HRRR modeling uses this sphere instead?

Thanks!

```
wgrib2 -code_table_3.2 hrrr.t06z.wrfprsf00.grib2
1:0:code table 3.2=6 Earth assumed spherical with radius = 6,371,229.0 m
2:706114:code table 3.2=6 Earth assumed spherical with radius = 6,371,229.0 m
3:1271749:code table 3.2=6 Earth assumed spherical with radius = 6,371,229.0 m
...
```

**Comment by EricJames-NOAA** [comment, marked as answer]
Hi @yosukefk, thanks for the question.  Unfortunately the GRIB2 encoding for the Earth radius is not very flexible, and there is no option for encoding the correct radius used in the model integration.  As you stated, WRF / HRRR uses 6370000 m, and the GRIB2 encoding is misleading.  So the correct value is 6370000 m.

Thanks,
Eric James
